### DISCUSSION TITLE
Does the AFS nio filesystem work well with distributed FUSE-based filesystem?

### DISCUSSION THREAD

**Discussion by vngantk** [microstream-one/microstream#276] [Q&A]
For fault tolerance purpose, I am considering mounting the nio FileSystem on a distributed file system mount point using distributed file systems like GlusterFS. Does MicroStream work well with this kind of setup?

**Comment by fh-ms** [comment]
We don't have any specific experiences with these file systems.
When you mount it, it should be fine since it is the operating system's job.
As an alternative, there's a NIO implementation for GlusterFS: https://github.com/gluster/glusterfs-java-filesystem
